src/modules/surjection/main_impl.h

```diff
@@ -221,8 +222,6 @@ int secp256k1_surjectionproof_generate(const secp256k1_context* ctx, secp256k1_s
     size_t ring_input_index = 0;
     secp256k1_gej ring_pubkeys[SECP256K1_SURJECTIONPROOF_MAX_N_INPUTS];
     secp256k1_scalar borromean_s[SECP256K1_SURJECTIONPROOF_MAX_N_INPUTS];
```

Comment:
Would it be possible to also change the above arrays' size to `SECP256K1_SURJECTIONPROOF_MAX_USED_INPUTS`?

Reply:
Yes. I don't know how I missed those.